server/database/connection.js

```diff
@@ -0,0 +1,18 @@
+const { Pool } = require('pg');
+const url = require('url');
+require('dotenv').config();
+
+const dbUrl = process.env.DB_URL;
```

Comment:
This is missing node environment settings for test and development database.

Reply:
I did it just fora test, I forgot to delete this file.